goentri.com.duda-subdomains.json

```diff
@@ -2,8 +2,8 @@
     "providerId": "goentri.com",
     "providerName": "Entri",
     "serviceId": "duda-subdomains",
-    "serviceName": "Third party application",
-    "version": 3,
```

Comment:
sure you want to go down with the version now?

Reply:
OK, that's fine. Tell me when you are done with the changes.
In the future please set the PR as Draft so that it's not being reviewed and merged before all commits are in.